src/Map.jsx

```diff
@@ -0,0 +1,89 @@
+import React, { Component } from 'react';
+import ReactMapGL, { NavigationControl } from 'react-map-gl';
+// import DeckGL, { GeoJsonLayer, IconLayer } from 'deck.gl';
+import {
+  graphql,
+  createFragmentContainer,
+} from 'react-relay';
+import propTypes from 'prop-types';
+import MAPBOX_ACCESS_TOKEN from './config';
```

Comment:
this should be './config.json' - you shouldn't be able to run it because of this.